config/locales/client.en.yml

```diff
@@ -1855,7 +1855,7 @@ en:
         same_as_username: "Your password is the same as your username."
         same_as_email: "Your password is the same as your email."
         ok: "Your password looks good."
-        instructions: "at least %{count} characters"
+        instructions: "At least %{count} characters."
```

Comment:
Sorry for posting the screenshot in the wrong place, noticed a a small inconsistency in styling here: 

<img width="222" alt="image" src="https://github.com/discourse/discourse/assets/368961/541fd796-12ff-4169-937a-af52adcd3ac3">

The first two descriptions' text color is different from the third.

Reply:
Weird but I can't repro 
<img width="459" alt="image" src="https://github.com/discourse/discourse/assets/101828855/c898051c-48f1-4e74-9dc6-addb810b33af">

Reply:
Hmm, can't repro today either.